OSD-2023.adoc

```diff
@@ -1,14 +1,13 @@
 ---
```

Comment:
Can we rename this to OSD.adoc? That way, we won't have to rename it again next year.

Reply:
fixed!